docs/component-guidance/inputter.md

```diff
@@ -0,0 +1,251 @@
+- To make forms usable with screen readers, Labels should always be used to ask the question that the input value answers. The exception being that Labels for Checkboxes and Radios are the answers to a posed question in the parent heading.
+
+---
+
+### Content
+
+- Inputs use native autocompletes. They will resize when using that attribute.
+- Spellcheck should be disabled for names within a text input and allow for accented characters, spaces and special characters such as hyphens, commonly found in people's names.
+- Consideration should be made for the right to left read order for internationalisation (eg. Arabic etc.) which can also affect the location of functional type icons within inputters.
+- Don't use invisible elements.
+- The input size correlates to its expected value, such as postcodes, telephone numbers and max length.
+
+---
+
+### Behaviour
+
+- Keyboard focus should tab through in the same order of appearance of the inputters.
+- Text entered into an inputter that is longer than the initial size of the field will behave like the browser's default.
+- The Search inputter will display a functional 'clear' icon at the opposite end to the search icon whilst text is being input, allowing the user to clear the contents if they wish.
+- Password should be hidden by default.
+- Clicking the functional icon on the Date should open the native datepicker calendar.
+- Textarea has a grab handle allowing the user to resize to view more of the text being entered but is  also scrollable.
+- Inputters default widths fit within the smallest viewport (320px wide)
+- By default inputter components stack in one column vertically and singly.
+- Radios and checkboxes are not selected by default leaving this choice to the user.
+
+---
+
+### Best practice
+
+#### Do
+
+- Only use the specified text type (eg. Text, Email etc.)
+- Use the autocomplete attribute
+- Base the width of input on the expected length of the input
+- Always include a label and keep them concise
+- Make labels easy to read and understand
+- Keep the heading/legend short
+- Associate every radio button with its label using ID & for attributes
+- Consider using a Select for more than 5 radio buttons (but remember this effectively hides options)
+- Use only one label with each checkbox
+- If adding a "Please Select" option then add an empty value="" attribute
+- Use headings to introduce a set of checkboxes or radio buttons
+- Keep Helper text to one line
+- Use Helper text to explain why info is needed
+- Use Helper text to clarify the meaning
+- Use Tip text for more context in conjunction with a helper, for explanation or as a prompt
+- Use Placeholder text to provide an example of the expected input into a field
+- Use Validation to assist the customer to identify errors when entering information and how they might correct it
+- Use combinations of validation types where required (`isRequired`, `isNumber`, etc.)
```

Comment:
```suggestion
- Use combinations of validation types where needed (`isRequired`, `isNumber`, etc.)
```

changed _required_ to _needed_ to avoid repetition and confusion with the meaning of the word that relates to the `isRequired` validation